src/main/java/com/mojang/brigadier/arguments/StringArgumentType.java

```diff
@@ -57,7 +84,16 @@ public String toString() {
 
     @Override
     public Collection<String> getExamples() {
-        return type.getExamples();
+        return type == StringType.TERM ? options : type.getExamples();
+    }
+
+    @Override
+    public <S> CompletableFuture<Suggestions> listSuggestions(final CommandContext<S> context, final SuggestionsBuilder builder) {
+        if (type != StringType.TERM) {
+            return Suggestions.empty();
+        }
+        options.stream().filter((s) -> s.startsWith(builder.getRemaining().toLowerCase())).sorted().forEachOrdered(builder::suggest);
```

Comment:
This assumes that the `options` values are all lowercase, but this is not guaranteed, is it?

Reply:
I agree. I will change it so that it matches behaviour of some other examples that Mojang uses (selection is case sensitive, suggestions - insensitive)